src/ray/raylet/local_object_manager.cc

```diff
@@ -315,10 +317,10 @@ void LocalObjectManager::AddSpilledUrls(
     auto it = objects_pending_spill_.find(object_id);
     RAY_CHECK(it != objects_pending_spill_.end());
 
+    spilled_objects_url_.emplace(object_id, object_url);
```

Comment:
Can you add a comment to explain why we must do this before sending to the directory?